pkg/auth/webapp/service2.go

```diff
@@ -332,16 +332,7 @@ func (s *Service2) doPost(
 		}
 	}
 
-	isCancelled := false
-	switch graph.CurrentNode().(type) {
-	case *nodes.NodeCancelInteraction:
-		isCancelled = true
-	}
-	if isCancelled {
-		err = nil
-	}
-
-	err = s.afterPost(result, session, graph, err, isCancelled, isFinished, isNewGraph)
+	err = s.afterPost(result, session, graph, err, isFinished, isNewGraph)
```

Comment:
It is okay for now. But if we want to revert some unwanted changes, it would be better to do it in a separate commit or even update the commit through rebase. So that we can have a clear commit history.